src/generators/cfml.ts

```diff
@@ -0,0 +1,113 @@
+import * as util from "../util.js";
+import type { Request } from "../util.js";
+
+import jsesc from "jsesc";
+
+const quote = (str: string): string => {
+  return jsesc(str, { quotes: "single" }).replace(/"/g, '""');
+};
+
+export const _toCFML = (request: Request): string => {
+  let cfmlCode = "";
+
+  cfmlCode += "httpService = new http();\n";
+  cfmlCode += 'httpService.setUrl("' + request.url + '");\n';
```

Comment:
You should quote anything because there's nothing stopping someone from passing in a URL with a quote in it to curl, even if that doesn't make sense.

```suggestion
  cfmlCode += 'httpService.setUrl("' + quote(request.url) + '");\n';
```

Reply:
good point i will wrap them

Reply:
Updated PR with `quote`